Doc/library/stdtypes.rst

```diff
@@ -3826,15 +3826,17 @@ another set.  The :class:`frozenset` type is immutable and :term:`hashable` ---
 its contents cannot be altered after it is created; it can therefore be used as
 a dictionary key or as an element of another set.
 
-Non-empty sets (not frozensets) can be created by placing a comma-separated list
-of elements within braces, for example: ``{'jack', 'sjoerd'}``, in addition to the
-:class:`set` constructor.
-
 The constructors for both classes work the same:
 
 .. class:: set([iterable])
            frozenset([iterable])
 
+   Sets can be created by several ways:
```

Comment:
'by several ways' is not idiomatic.  Either 'by several means' or 'in several ways'.  The latter is used below for dicts.